pkg/apis/core/helper/helpers.go

```diff
@@ -483,3 +484,16 @@ func IsMultiZonalSeed(seed *core.Seed) bool {
 	}
 	return seed.Spec.HighAvailability != nil && seed.Spec.HighAvailability.FailureTolerance.Type == core.FailureToleranceTypeZone
 }
+
+// ValidateMultiZoneSeedLabelValue validates the value of label `v1beta1constants.LabelSeedMultiZonal`.
+func ValidateMultiZoneSeedLabelValue(val string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if len(val) > 0 {
```

Comment:
Nit: Typically we try to exit early to reduce the nesting, thus improving readability:
```suggestion
	if len(val) == 0 {
		return allErrs
	}

	// ...
```

Reply:
changed. PTAL